SegmentEditorSplitVolume/SegmentEditorSplitVolumeLib/SegmentEditorEffect.py

```diff
@@ -57,10 +58,22 @@ def updateGUIFromMRML(self):
     except:
       self.padEdit.setValue(5)
     self.padEdit.blockSignals(wasBlocked)
+    
+    wasBlocked = self.applyToAllVisibleSegmentsCheckBox.blockSignals(True)
+    try:
+      checked = True if (self.scriptedEffect.integerParameter("ApplyToAllVisibleSegments") == 1) else False
+      self.applyToAllVisibleSegmentsCheckBox.setChecked(checked)
+    except:
+      self.applyToAllVisibleSegmentsCheckBox.setChecked(True)

```

Comment:
When does this exception can happen?
`integerParameter` returns 0 if the the parameter is not found. If you want to ensure that the option is enabled by default when the parameter is not defined then you can check `parameter(...)` and if that returns an empty string it means that the parameter is not defined and you can set the checkbox checked.